packages/store/src/StoreCore.sol

```diff
@@ -572,12 +572,49 @@ library StoreCore {
     FieldLayout fieldLayout
   ) internal view returns (bytes memory) {
     if (fieldIndex < fieldLayout.numStaticFields()) {
-      return StoreCoreInternal._getStaticField(tableId, keyTuple, fieldIndex, fieldLayout);
+      return StoreCoreInternal._getStaticFieldBytes(tableId, keyTuple, fieldIndex, fieldLayout);
```

Comment:
We still need a method that returns the field as dynamic `bytes memory` with the correct length here